doc/files.am

```diff
@@ -953,6 +953,7 @@ absolute_source_files = \
 	$(top_srcdir)/doc/source/install/solaris.rst \
 	$(top_srcdir)/doc/source/install/ubuntu.rst \
 	$(top_srcdir)/doc/source/install/windows.rst \
+	$(top_srcdir)/doc/source/install/cmake.rst \
```

Comment:
Did you use `make -C doc update-files` to update this list?

Reply:
Ah, No. I didn't know that but I found it is written in release procedure.(No need to update the document.)
Thanks!!

Reply:
I have executed `make -C doc update-files`.
Some unrelated files are added to `files.am` but they may be necessary.